test/functional/legacy/025_jump_tag_hidden_spec.lua

```diff
@@ -39,7 +42,12 @@ describe('jump to a tag with hidden set', function()
     -- space will then be eaten by hit-return, instead of moving the cursor to 'd'.
     feed_command('set tags=tags.test')
     feed('G<C-]> x:yank a<cr>')
-    feed_command('!rm -f Xxx test25.dir tags.test')
+    if helpers.iswin() then
+      feed_command('!del /q/f Xxx tags.test')
+      feed_command('!rd /q test25.dir')
```

Comment:
`helpers.rmdir()`. Can `command("call delete('tags.test')"` be used for the file?

Reply:
Yes.